test.cpp

```
@@ -0,0 +1,135 @@
+        cout << "Grade: " << students[i].grade << "\n";
+        cout << "Gender: " << students[i].gender << "\n";
+        cout << "----------------------\n";
+    }
+}
+
+float calculateAverage(Student* students, int count) {
+    float sum;
+    for (int i = 0; i < count; i++) {
+        sum += students[i].grade; // 錯誤：sum 未初始化
+    }
+    return sum / count; // 邏輯錯誤：若 count = 0，會除以 0
```

Cppcheck (2.10-2)

[warning] Uninitialized variable

(uninitvar)